build.gradle.kts

```
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2022 Exactpro Systems Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    `java-gradle-plugin`
+    id("maven-publish")
+}
+
+repositories {
+    gradlePluginPortal()
+}
+
+// Kotlin 1.4 is the last version supporting Kotlin API 1.2 which is bundled with Corda 4
+val kotlinVersion = "1.4.32"
+
+dependencies {
+    implementation(gradlePlugin("org.jetbrains.kotlin.jvm", kotlinVersion))
+    // Kotlin style checker (3.0.x is for Kotlin 1.4)
+    implementation(gradlePlugin("org.jmailen.kotlinter", "3.0.2"))
+    // Documentation engine for Kotlin
+    implementation(gradlePlugin("org.jetbrains.dokka", kotlinVersion))
+}
+
+configurations.all {
+    resolutionStrategy.dependencySubstitution {
+        substituteKotlinModule("org.jetbrains.kotlin:kotlin-stdlib-jdk7")
+        substituteKotlinModule("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    }
+}
+
+gradlePlugin {
+    plugins {
+        create("kotlinCommon") {
+            id = "com.gevamu.kotlin-common-conventions"
+            implementationClass = "com.gevamu.KotlinCommonPlugin"
+        }
+    }
+}
+
+fun gradlePlugin(id: String, version: String) = "$id:$id.gradle.plugin:$version"
+
+fun DependencySubstitutions.substituteKotlinModule(moduleNotation: String) {
+    substitute(module(moduleNotation)).using(module("$moduleNotation:$kotlinVersion"))
+}
+
+publishing {
+    publications {
+        create<MavenPublication>("kotlin-common-conventions") {
+            artifactId = "gevamu-kotlin-common-conventions"
+            pom {
+                name.set("Gevamu kotlin common conventions")
+                url.set("https://github.com/gevamu/kotlin-common-conventions")
+                licenses {
+                    license {
+                        name.set("The Apache License, Version 2.0")
+                        url.set("https://www.apache.org/licenses/LICENSE-2.0.txt")
+                    }
+                }
+                scm {
+                    connection.set("scm:git:git://github.com/gevamu/kotlin-common-conventions.git")
+                    developerConnection.set("scm:git:git://github.com/gevamu/kotlin-common-conventions.git")
```

This is my mistake, I copy-pasted it from another project
The correct way to have these links set up is `scm:git:https://github.com/...`
```suggestion
                    connection.set("scm:git:https://github.com/gevamu/kotlin-common-conventions.git")
                    developerConnection.set("scm:git:https://github.com/gevamu/kotlin-common-conventions.git")
```